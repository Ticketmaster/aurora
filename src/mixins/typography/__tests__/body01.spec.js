import BODY_01 from "../body01";

describe("BODY_01", () => {
  it("should return the correct values", () => {
    expect(BODY_01).toMatchSnapshot();
  });
});
