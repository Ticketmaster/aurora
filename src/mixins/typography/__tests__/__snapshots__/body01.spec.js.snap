// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BODY_01 should return the correct values 1`] = `
Array [
  "
  ",
  "
  color: ",
  [Function],
  ";
",
  ";
  ",
  "
  font-size: ",
  "16px",
  ";
",
  ";
  ",
  "
  line-height: ",
  "1.5",
  ";
",
  ";
",
]
`;
